Description: Jenkins 1.409.x upstream uses an older version of stapler and
 the maven-stapler-plugin that those present in Debian and Ubuntu.  As a result
 the packaging for Jenkins core needs to be updated to be a straight jar as the
 newer version of stapler uses JSR-269 for annotation processing and hooks into
 the standard compile phase of maven.
 .
 This patch can be dropped when Jenkins release their next LTS release.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

--- a/core/pom.xml
+++ b/core/pom.xml
@@ -652,6 +652,24 @@ THE SOFTWARE.
         </executions>
       </plugin>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>2.3.2</version>
+        <configuration>
+          <excludes>
+             <exclude>**/os/solaris/*</exclude>
+             <exclude>**/os/windows/*</exclude>
+             <exclude>**/DCOMSandbox.java</exclude>
+             <exclude>**/DotNet.java</exclude>
+             <exclude>**/WindowsSlaveInstaller.java</exclude>
+             <exclude>**/WindowsInstallerLink.java</exclude>
+          </excludes>
+          <testExcludes>
+             <exclude>**/DCOMSandbox.java</exclude>
+          </testExcludes>
+        </configuration>
+      </plugin>
+      <plugin>
         <groupId>org.kohsuke.stapler</groupId>
         <artifactId>maven-stapler-plugin</artifactId>
         <version>1.16</version>
